web/src/utils/exporter.js

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import font from '!url-loader?limit=undefined!@openfonts/barlow-condensed_all/files/barlow-condensed-all-400.woff2';
```

Comment:
Can we put a loader rule into `vue.config.js` instead of doing it inline?

Reply:
I need to force the loader to inline. By default it would fallback to the file-loader since it would consider the font too large to inline